### ISSUE TITLE
[bug] Group convolution error when group=2 in pytorch 0.4.0

### ISSUE THREAD

**Issue by seongwook-ham** [pytorch/pytorch#7281]
In pytorch0.4.

i use torch.nn.DataParallel for 4 gpu
when featuremap size is [128, 106, 32, 32], kernel_size=3, group=2, padding=1 in_channels=106, out_channels= 106
checked for stride=2 and 1

torch.nn.Conv2d(106, 106, 3, stride=1, padding=1, dilation=1, groups=2, bias=False)

torch.nn.Conv2d(106, 106, 3, stride=2, padding=1, dilation=1, groups=2, bias=False)

splited featuremap size on each gpu is
torch.Size([32, 106, 32, 32])

misaligned address error is printed

i don’t know why this happens
other size with group=2 works well

i have enough gpu  memory for those operation. 4 gtx 1080ti

**Comment by li-roy** [comment]
I'm having trouble reproducing your error, can you please provide a code sample that can reproduce it?

**Comment by soumith** [comment]
We haven't heard back from you in a while, please feel free to open once you give us a code sample.
